src/App.tsx

```diff
@@ -1,24 +1,136 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as todosService from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Errors } from './types/Errors';
+import { FilterValue } from './types/FilterValue';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 12037;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState(FilterValue.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.Null);
+  const [loadingTodoId, setLoadingTodoId] = useState<number[]>([]);
+
+  const handleError = (error: Errors) => {
+    setErrorMessage(error);
+    setTimeout(() => setErrorMessage(error), 3000);
```

Comment:
```suggestion
    setTimeout(() => setErrorMessage(Errors.Null), 3000);
```
In setTimeout, you should reset the error